.travis.yml

```diff
@@ -60,6 +60,26 @@ jobs:
         MODEL_TAG_NAME: sentence-transformers-gtr-t5-large
       if: tag IS present
       stage: buildanddeploy
+    - env:
+        MODEL_NAME: vblagoje/dpr-ctx_encoder-single-lfqa-wiki
+        MODEL_TAG_NAME: vblagoje-dpr-ctx_encoder-single-lfqa-wiki
+      if: tag IS present
+      stage: buildanddeploy
+    - env:
+        MODEL_NAME: vblagoje/dpr-question_encoder-single-lfqa-base
```

Comment:
@antas-marcin here we also use wiki not base

Reply:
I have fixed that, thanks for checking this @vblagoje